Tasks/DownloadBuildArtifactsV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -17,12 +17,14 @@
   "loc.input.help.branchName": "Specify to filter on branch/ref name, for example: ```refs/heads/develop```.",
   "loc.input.label.buildId": "Build",
   "loc.input.help.buildId": "The build from which to download the artifacts",
+  "loc.input.label.Tags": "Build Tags",
+  "loc.input.help.Tags": "A comma-delimited list of tags. Only builds with these tags will be returned.",
   "loc.input.label.downloadType": "Download type",
   "loc.input.help.downloadType": "Download a specific artifact or specific files from the build.",
   "loc.input.label.artifactName": "Artifact name",
   "loc.input.help.artifactName": "The name of the artifact to download",
   "loc.input.label.itemPattern": "Matching pattern",
-  "loc.input.help.itemPattern": "Specify files to be downloaded as multi line minimatch pattern. [More Information](https://aka.ms/minimatchexamples) <p>The default pattern (\\*\\*) will download all files across all artifacts in the build. To download all files within artifact drop use drop/**.</p>",
+  "loc.input.help.itemPattern": "Specify files to be downloaded as multi line minimatch pattern. [More Information](https://aka.ms/minimatchexamples) <p>The default pattern (\\*\\*) will download all files across all artifacts in the build if \"Specific files\" option is selected. To download all files within artifact drop use drop/**.</p>",
```

Comment:
This was checked in on May 17th. How can I resolve this conflict?

Reply:
You can pick up the checked in version of this message.